src/modules/AnalysisExecution/const.js

```diff
@@ -197,20 +198,22 @@ const submissionActionTypes = keyMirror({
   HIDE: null,
 });
 
-const importableAnalysisTypes = ['COHORT', 'ESTIMATION', 'PREDICTION', 'COHORT_CHARACTERIZATION', 'INCIDENCE'];
+const importableAnalysisTypes = ['COHORT', 'ESTIMATION', 'PREDICTION', 'COHORT_HERACLES', 'COHORT_CHARACTERIZATION', 'INCIDENCE'];
 const analysisTypeNames = {
   COHORT: 'cohort',
   ESTIMATION: 'PLE analysis',
   PREDICTION: 'PLP analysis',
-  COHORT_CHARACTERIZATION: 'cohort',
+  COHORT_HERACLES: 'cohort',
```

Comment:
Why not cohort Heracles analysis? or smth like this? - where is the diff with the regular `cohort`?